samples/matter/lock/src/lit/zap-generated/callback-stub.cpp

```diff
@@ -79,9 +79,6 @@ void emberAfClusterInitCallback(EndpointId endpoint, ClusterId clusterId)
 	case app::Clusters::ThreadNetworkDiagnostics::Id:
 		emberAfThreadNetworkDiagnosticsClusterInitCallback(endpoint);
 		break;
-	case app::Clusters::WiFiNetworkDiagnostics::Id:
```

Comment:
Did you remove WiFiNetworkDiagnostics purposely? I don't see such change to the SIT configuration.

Reply:
Right I need to change that after for sit too,
since the test cases verifies the conformance of the PICS with the supported cluster I need to remove that for Test event purposes